### ISSUE TITLE
on-screen keyboard not displayed after boot until hardware keyboard has been opened once

### ISSUE THREAD

**Issue by schmittlauch** [sailfish-on-fxtecpro1/droid-config-t5#38]
The virtual on-screen keyboard is not displayed after boot until the hardware keyboard has been opened and closed once.
This is a known issue, just adding it here for completeness sake.

release: 3.3.0.16

**Comment by piggz** [comment]
Does android have similar behaviour?

**Comment by Kabouik** [comment]
No, the virtual keyboard works immediately in LineageOS according to another user. Maybe the HAL sensor doesn't start in the same state in SFOS and is reset with the first open/close?

**Comment by piggz** [comment]
If the keyboard is opened while booting android, does it show the onscreen kbd or use the hw keyboard?

**Comment by Kabouik** [comment]
> 09:28] kabouik: Another question from the porter: what happens if the keyboard is open while booting? Does the virtual keyboard show up too?
> [13:12] Allury: I'll check
> [13:16] Allury: During boot the numeric vtkb shows up. When the phone is booted and I now opened discord both hwkb and vtkb are present
> [13:17] Allury: Closing and reopening hwkb makes the vtkb disappear

**Comment by piggz** [comment]
So, it seems the same issue exists in android, but they default to having the VKB enabled, and we default to not ?

**Comment by Kabouik** [comment]
Looks like it yes, but that seems less problematic in their case because at worst they have vkb and hkb working simultaneously, but only when booting with keyboard open, which might be a bit more of a corner case.

**Comment by piggz** [comment]
sure, im just trying to find out if there is a  way to change the default on sfos

**Comment by Kabouik** [comment]
A different report from a LineageOS user:

> I don't get this with Lineage.  I don't think I ever booted with the keyboard out on stock, so don't know.  In lineage, if I boot with KB out, as soon as it  gets to lockscreen, the screen goes landscape and I never see a VKB.  If I close the keyboard and go portrait, the VKB pops up as appropriate.  This is using AOSP  VKB
